server/server.go

```diff
@@ -42,7 +42,7 @@ func Start(ctx context.Context, httpPort, httpsPort int, scaledContext *config.S
 
 	k8sProxy := k8sProxyPkg.New(scaledContext, scaledContext.Dialer)
 
-	managementAPI, err := managementapi.New(ctx, scaledContext, clusterManager, k8sProxy)
+	managementAPI, err := managementapi.New(ctx, scaledContext, clusterManager, k8sProxy, addLocal)
```

Comment:
nit: can we use more descriptive name for addLocal in this context? Something that would indicate it means that local cluster is being added.